examples/date_and_time/README.rst

```diff
@@ -0,0 +1,14 @@
+Date Picker Example
+===================
+
+Test app for the Date Picker Example widget.
+
+Date Picker is available only on Android and Win platforms
```

Comment:
This README is now out of date - that's my fault for not corrected it when I renamed the example, but it needs to be corrected before merging.

Reply:
This is done now